packages/components/cascader/src/cascader.ts

```diff
@@ -0,0 +1,79 @@
+import { CommonProps } from '@element-plus/components/cascader-panel'
+import { isBoolean, isValidComponentSize } from '@element-plus/utils'
+import { useTooltipContentProps } from '@element-plus/components/tooltip'
+import { tagProps } from '@element-plus/components/tag'
+import { CHANGE_EVENT, UPDATE_MODEL_EVENT } from '@element-plus/constants'
+import type { ComponentSize } from '@element-plus/constants'
+import type { PropType } from 'vue'
+import type {
+  CascaderNode,
+  CascaderValue,
+} from '@element-plus/components/cascader-panel'
+
+export const cascaderProps = {
```

Comment:
## Question

Is there a reason for not using `buildProps`?

Reply:
well.